installer/registry.go

```diff
@@ -47,7 +49,7 @@ func (r *registry) AddBundleInstaller(os, k8sVer string, installer osk8sInstalle
 		panic(fmt.Sprintf("%v %v already exists", os, k8sVer))
 	}
 
-	r.osk8sInstallerMap[os][k8sVer] = installer
+	r.osk8sInstallerMap[os][k8sVer] = empty
```

Comment:
is this required? as we are already initializing `osk8sInstallerMap` in `newRegistry` func(`make(osk8sInstallerMap)` will create an empty map), which is called before `AddBundleInstaller`

Reply:
The make() will give us an empty map with no keys or values. Just to add keys to this support matrix we use empty interfaces. Example: https://go.dev/play/p/_MiQobRR64o